srt-tokio/tests/stransmit_interop.rs

```diff
@@ -154,6 +154,19 @@ async fn stransmit_client() -> Result<(), Error> {
     listener.await;
     stransmit.wait()?;
 
+    if time::Instant::now().elapsed() > Duration::MAX {
```

Comment:
I still don't quite understand this. How is this ever true? This seems like just a roundabout way to disable some code?

Reply:
Yeah, this can be removed. These tests weren't running correctly on the mac, but the compiler was complaining about dead code. You'll probably also have to disable the test for mac os though. I thought I added a comment to the code for this, but obviously didn't...

Reply:
We can probably add a `#[allow(dead_code)]`, unless it is many many things that would need marking

Reply:
Well, yes, I started there, but the complication is that there are multiple things that would need to be annotated, including static const, struct, and fn. I probably could have figured it all out, I was being expedient. Perhaps all the linux specific non-test code should be in a submodule, which can be annotated accordingly too?

Reply:
Well the *ideal* solution would be to get that test to work on macOS. We do install srt on mac via brew, so there's nothing technically in the way--since you have a mac it would be great if you could investigate it? (Would be another PR, and we could leave this as-is for now)

Reply:
Ok, I can do that.